docs/configure-artifact-repository.md

```diff
@@ -344,6 +344,10 @@ Create an Azure Storage account and a container within that account. There are a
 ways to accomplish this, including the [Azure Portal](https://portal.azure.com) or the
 [CLI](https://docs.microsoft.com/en-us/cli/azure/).
 
+There are two ways to allow argo to authenticate it's access to your Azure storage account.
+
+### Using Azure access keys
```

Comment:
This makes it seem like it's either access key or SAS. However, there's also the `useSDKCreds` option where you don't need a secret set and it will use a `DefaultAzureCredential` which supports a number of methods of authentication. I would call that option out first, since it's probably the most secure and preferred auth method. Then access key and SAS are the other two options.

Reply:
Yes that makes sense! Let me push a change to the doc!